### ISSUE TITLE
Get data on # of actions per typical campaign email

### ISSUE THREAD

**Issue by adamlofting** [MozillaFoundation/encryption_campaign#29]
I.e. if we mail our list, how many people do we think will take an action.

Then, set a target for growth.

We'll use this to inform our Feb and Mar goals for number of active advocates.

Assigned to @valianttry in case there is some time to start on this today. Also Steph is looking at some of this data this week.

**Comment by OpenMatt** [comment]
- @stephaniemcv @valianttry any updates from work week to include here?

**Comment by OpenMatt** [comment]
@valianttry not sure what's needed this week to close this ticket.

**Comment by valianttry** [comment]
@OpenMatt I just reassigned this to @stephaniemcv -- would you mind double checking she has her Github set up to get alerts when she's tagged? (I want to make sure she's not missing stuff)

**Comment by stephaniemcv** [comment]
I got an email notification. ^_^ I have a bug submitted for view access on GA for different Advocacy campaigns. Once that is finished I'll be able to report back on numbers.

**Comment by OpenMatt** [comment]
@stephaniemcv cool. rolling this over to next week.

**Comment by OpenMatt** [comment]
@stephaniemcv rolling this over to next week.

**Comment by stephaniemcv** [comment]
Numbers based on 2015 Advocacy Email data

Average Open Rate: 21.69%
Median Open Rate: 14.33%

Average Click Rate: 4.61%
Median Click Rate: 2.58%

Average Action Rate: 2.06%
Median Action Rate: 1%

**Comment by adamlofting** [comment]
Another slice of this data that I looked at for the board meeting prep was an 'Action Rate per campaign' (rather than per email).

For the two biggest actions, Net Neutrality and Section 215 we saw ~3.5% of the existing list taking action via email. This was the combined result of sending several asks to the same list. The individual emails had a lower action rate, but repetition reached a greater % of the list overall.
